lib/api/core/auth/formatProcessing.js

```diff
@@ -61,34 +36,11 @@ module.exports = {
    * @param {boolean} hydrate
    * @returns {Object}
    */
-  formatUserForSerialization: function (kuzzle, user, hydrate) {
+  formatUserForSerialization: function (kuzzle, user) {
     return kuzzle.pluginsManager.trigger('security:formatUserForSerialization', user)
       .then(triggeredUser => {
-        if (typeof triggeredUser.profile === 'string') {
-          return kuzzle.repositories.profile.loadProfile(triggeredUser.profile)
-            .then(profile => {
-              triggeredUser.profile = profile;
-              return q(triggeredUser);
-            });
-        }
-
-        return q(triggeredUser);
-      })
-      .then(triggeredUser => {
-        var
-          response = triggeredUser;
-
-        if (hydrate) {
-          response = {_id: triggeredUser._id, _source: {}};
-          Object.keys(triggeredUser).forEach((key) => {
-            if (key === 'profile') {
-              response._source.profile = this.formatProfileForSerialization(triggeredUser.profile, hydrate);
-            }
-            else if (key !== '_id') {
-              response._source[key] = triggeredUser[key];
-            }
-          });
-        }
+        var response = _.assignIn({}, {_id: triggeredUser._id, _source: triggeredUser});
```

Comment:
Same than above: wouldn't it be simpler to write `var response =  {_id: triggeredUser._id, _source: triggeredUser};`?